paket.dependencies

```diff
@@ -1,14 +1,20 @@
 source https://nuget.org/api/v2
 
 nuget FSharp.Core prerelease redirects: force
-nuget Fable.Core
 nuget Fable.JsonConverter prerelease
 nuget Suave
 nuget Microsoft.Extensions.Logging.Console
 nuget System.Net.NetworkInformation
 nuget Newtonsoft.Json prerelease
 nuget jose-jwt
 
+# Keep this version synced with dotnet-fable in .fsproj
+nuget Fable.Core 1.0.4
```

Comment:
I thought the comment was self-explanatory ;) How else can you be sure that `paket install` will install the same version specified for dotnet-fable in the .fsproj?